docker/dockerfile-alpine

```diff
@@ -1,18 +1,31 @@
-FROM louislam/uptime-kuma:base-alpine AS build
+FROM louislam/uptime-kuma:base-alpine AS build_server
```

Comment:
I think that support for alpine got dropped a while ago because it was too much work to maintain two seperate dockerfiles

Reply:
Yeah i saw that somewhere, understandable but also a shame

Reply:
No reason to not try to update it though, since builds are still being published to Dockerhub at least for the moment. I am building uptime-kuma in Alpine natively and might push it to Alpine repos at a later date, Docker would be even easier then.